### PR TITLE
Add support for publishing gem to GPR

### PR DESCRIPTION
**Pull request by ijdickinson**
This commit addresses epimorphics/hmlr-linked-data#88 by adding support,
via a `Makefile` for publishing this gem to the Epimorphics section of
the Github Package Registry (GPR)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.PHONY: auth clean gem publish test
+
+NAME?=lr_common_styles
+OWNER?=epimorphics
+VERSION?=$(shell ruby -e 'require "./lib/lr_common_styles/version" ; puts LrCommonStyles::VERSION')
+PAT?=$(shell read -p 'Github access token:' TOKEN; echo $$TOKEN)
```
[the remark this answers is not in the document]
Comment (ijdickinson) [reply Makefile:6]: Good point. I'll add a section to the docs

[the remark this answers is not in the document]
Comment (ijdickinson) [reply Makefile:6]: Done

**Review by jonrandahl — APPROVED** [review]
LGTM! 😄 

QQ: Will this resolve the outstanding dependencies on the repo? 🤔

**Review by bogdanadrianmarc — APPROVED** [review]
Changes look good 👍

**Pull request merged**